src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderBooking.java

```diff
@@ -109,7 +109,6 @@ public class OffenderBooking extends AuditableEntity {
     @Exclude
     private List<CourtOrder> courtOrders = new ArrayList<>();
 
-    @ListIndexBase(1)
```

Comment:
this annotation looks like it was left on after a copy from a different field. Doesn't make any sense without an order by but there isn't a sequence on this field anyway and therefore hibernate generated invalid sql.